Project.toml

```diff
@@ -3,9 +3,11 @@ uuid = "7a12625a-238d-50fd-b39a-03d52299707e"
 version = "3.6.1"
 
 [deps]
+ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
```

Comment:
FYI this package only depends on `Compat`, `SparseArrays` and `LinearAlgebra` so it's quite small.

Reply:
Yea, the only issue seems to be that it adds quite a number of (potential) method invalidations, but I guess this is something people are working on over there.